DOCS-#3098: Adding Intel distribution of Modin section (#3312)
modin-project/modin: docs/installation.rst

  conda install -c conda-forge modin-ray modin-dask modin-omnisci

Using Intel\ |reg| Distribution of Modin
"""""""""""""""""""""""""""""""""""""""

With ``conda`` it is possible to install special version of Modin, part of Intel\ |reg| oneAPI AI Analytics Toolkit.

> This version of Modin powered by :doc:`OmniSci</UsingOmnisci/index>` engine that contains a bunch of

Reviewer: ```suggestion
This version of Modin is powered by :doc:`OmniSci</UsingOmnisci/index>` engine that contains a bunch of
```

> This version of Modin powered by :doc:`OmniSci</UsingOmnisci/index>` backend that contains a bunch of

Reviewer 2: ```suggestion
This version of Modin powered by :doc:`OmniSci</UsingOmnisci/index>` engine that contains a bunch of
```

> optimizations for Intel hardware. All details can be found here_.

Reviewer 2: ```suggestion
optimizations for Intel hardware. All details can be found on `Intel Distribution of Modin`_ page.
```

.. _Ray: http://ray.readthedocs.io
.. _Dask: https://github.com/dask/dask
.. _OmniSci: https://www.omnisci.com/platform/omniscidb

> .. _here: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-of-modin.html#gs.86stqv

Reviewer 2: ```suggestion
.. _`Intel Distribution of Modin`: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-of-modin.html#gs.86stqv
```
